build/src/standalone.js

```diff
@@ -0,0 +1,12 @@
+export * from "./content_tag.js";
+
+import init, { Preprocessor } from "./content_tag.js";
+
+export async function createPreprocessor() {
```

Comment:
just a tiny wrapper function since the default way rust wants us to build a wasm thing is ... goofy. Like, I get it, two things need to happen, load the wasm file and then instantiate the stuff.
maybe I should PR to them.